source/val/validation_state.h

```diff
@@ -820,6 +833,10 @@ class ValidationState_t {
   std::unordered_map<uint32_t, std::vector<Instruction*>>
       sampled_image_consumers_;
 
+  /// Stores load instructions that load texturesed used
+  //  in QCOM image processing functions
+  std::unordered_set<uint32_t> tpip_texture_consumers_;
```

Comment:
I'm not sure this is a good variable name, what is `tpip`? Maybe `qcom_image_processing_consumers` would be better.

Reply:
Done